server/src/projects/projects.service.ts

```diff
@@ -102,6 +101,11 @@ export class ProjectsService {
       `);
 
       const [ project ] = this.overwriteCodesWithLabels(records);
+
+      if (!project) {
+        throw 'No projects found.';
```

Comment:
Can we throw this with code, title, detail, status? 
For consistency for now. If we redesign the exception handling, it'll be easier to find and replace.

Reply:
okay!